src/components/shared/UpcomingActions/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './index.scss';
+
+type UpcomingActionsProps = {
+  children: any;
+  timestamp: string;
+};
+
+const UpcomingActions = ({ children, timestamp }: UpcomingActionsProps) => {
+  return (
+    <div className="upcoming-actions">
+      <span className="upcoming-actions__header">
+        <h1>Upcoming Actions</h1>
+        <p>as of {timestamp}</p>
```

Comment:
`<p />` can be probably be changed to a `<span />` because the CSS is adding `display: inline`. `<span />` is an inline element.